src/agner_health.erl

```diff
@@ -2,9 +2,9 @@
 
 -export([init/2]).
 
-init(Req0 = #{method := <<"GET">>}, State) ->
-  Req = cowboy_req:reply(200,
+init(Req = #{method := <<"GET">>}, State) ->
+  Res = cowboy_req:reply(200,
```

Comment:
https://ninenines.eu/docs/en/cowboy/2.0/manual/cowboy_req.reply/

A new Req object is returned.

The returned Req object should be used from that point onward as it contains updated information about the state of the request.

Reply:
Jak pierwszy raz wszedłem w ten plik to się zdziwiłem czemu odpowiedzią na request jest inny request :smile: 

Widzę że tak jest nawet w docku dla samego responsu: https://ninenines.eu/docs/en/cowboy/1.0/guide/resp/

Ale jak dla mnie odpowiedzią na request jest response stąd ta nazwa - nawet jeśli de fakto reply zwraca odpowiedź na request + sam request (?)